src/ViewModels/ArcGisMapServerItemViewModel.js

```diff
@@ -66,6 +66,8 @@ ArcGisMapServerItemViewModel.prototype._enableInCesium = function() {
 
     var scene = this.application.cesium.scene;
 
+    this.legendUrl = this.url + '/legend';
```

Comment:
I've dropped legendUrl from the init file and instead figured out what the 'right' url should be in all our random cases so we can generate it based on the service.  I want to hopefully stay out of that business and let the providers set their servers up properly.  We can always add it back later if someone gives us a sensible reason.